### ISSUE TITLE
Idea: impl FromStr for MaskedIpAddr

### ISSUE THREAD

**Issue by danneu** [nuew/postgres-inet#3]
It'd be nice for the core struct of this lib to impl FromStr.

```rust
let a: MaskedIpAddr = "1.2.3.4/24".parse().unwrap();
let b: MaskedIpAddr = "1.2.3.4".parse().unwrap(); // prefix is 32 
```

**Comment by danneu** [comment]
Oops

**Comment by danneu** [comment]
Another idea is to wrap https://docs.rs/ipnetwork/0.12.7/ipnetwork/

**Comment by nuew** [comment]
Sorry about taking so long to finish this... I've been busy with school.

Both of these requests are implemented with 0.15.0.

**Comment by danneu** [comment]
Great stuff, thanks
